cli/src/commands/duplicate.rs

```diff
@@ -58,14 +60,10 @@ pub(crate) fn cmd_duplicate(
     let store = base_repo.store();
     let mut_repo = tx.mut_repo();
 
-    for original_commit_id in base_repo
-        .index()
-        .topo_order(&mut to_duplicate.iter().map(|c| c.id()))
-        .into_iter()
-    {
+    for original_commit_id in to_duplicate.iter().rev() {
         // Topological order ensures that any parents of `original_commit` are
```

Comment:
I only looked at this briefly, and I'm wondering: is this going to still be true? What ensures that children are duplicated after parents?

If `parse_union_revsets` produces revisions in topological order, this should be documented.

It's also very possible that I missed something.

Reply:
It's documented in `Revset::iter()`, but I added inline comment for clarity.